src/.env.example

```diff
@@ -1,2 +1,7 @@
+##### APP #####
 PORT="APP_PORT"
-ADMIN="ADMIN_PASSWORD"
\ No newline at end of file
+ADMIN="ADMIN_PASSWORD"
+
+##### Provider API Keys #####
+API_KEY_GO_DADDY="API_KEY_GO_DADDY"
+SECRET_GO_DADDY="SECRET_GO_DADDY"
```

Comment:
could remove. Grab it from the API

Reply:
I did but the `db` file do not exist and `ServiceKey` type is not define within `data.ds`